calendar-api.js

```diff
@@ -91,6 +91,119 @@ module.exports = {
   },
   
   /**
+   * Syncs events of a calendar.
+   * 
+   * @param parameters {object} An object containing all of the relevant parameters. Possible values:
+   * @param parameters.token {string} The access token.
+   * @param [parameters.skipToken] {string} The value to pass in the `skipToken` query parameter in the API call.
+   * @param [parameters.deltaToken] {string} The value to pass in the `deltaToken` query parameter in the API call.
+   * @param [parameters.useMe] {boolean} If true, use the `/Me` segment instead of the `/Users/<email>` segment. This parameter defaults to false and is ignored if the `parameters.user.email` parameter isn't provided (the `/Me` segment is always used in this case).
+   * @param [parameters.user.email] {string} The SMTP address of the user. If absent, the `/Me` segment is used in the API URL.
+   * @param [parameters.user.timezone] {string} The timezone of the user.
+   * @param [parameters.calendarId] {string} The calendar id. If absent, the API calls the `/User/calendarview` endpoint. Valid values of this parameter are:
+   * 
+   * - The `Id` property of a `Calendar` entity
+   * - `Primary`, the primary calendar is used. It is used by default if no Id is specified
+   * 
+   * @param [callback] {function} A callback function that is called when the function completes. It should have the signature `function (error, result)`.
+   * 
+   * @example var outlook = require('node-outlook');
+   * 
+   * // Set the API endpoint to use the 2.0 version of the api
+   * outlook.base.setApiEndpoint('https://outlook.office.com/api/v2.0');
+   * 
+   * // This is the oAuth token 
+   * var token = 'eyJ0eXAiOiJKV1Q...';
+   * 
+   * // Pass the user's email address
+   * var userInfo = {
+   *   email: 'sarad@contoso.com'
+   * };
+   *
+   * // You have to specify a time window
+   * var startDateTime = "2017-01-01";
+   * var endDateTime = "2017-12-31";
+   * 
+   * var apiOptions = {
+   *   token: token,
+   *   calendarId: 'Primary', // Not mandatory, the Primary calendar will be used by default, but you can specify another calendar Id
```

Comment:
You're right again, I was thinking about Google calendar API, but it does not seem to exist with Outlook API.